www/i18n/en.json

```diff
@@ -380,7 +381,8 @@
     "all-green-status": "Make sure that all status checks are green",
     "dont-force-kill": "Do not force kill the app",
     "background-restrictions": "On Samsung and Huwaei phones, make sure that background restrictions are turned off",
-    "close": "Close"
+    "close": "Close",
+    "open-access-study": "Link to Open Access Study"
```

Comment:
You can remove this - such a link is not planned at this time.

Reply:
Removed in https://github.com/e-mission/e-mission-phone/pull/1118/commits/bd9b2f2bd55c94c16bc4416db28529b435e2c028